docs/CMakeLists.txt

```diff
@@ -47,6 +47,16 @@ configure_file(
     "${BINARY_BUILD_DIR}/conf.py"
     @ONLY)
 
+add_custom_target(spell-check
+    COMMAND /bin/bash "${CMAKE_CURRENT_SOURCE_DIR}/check_spelling.sh" "-q" "${CMAKE_CURRENT_SOURCE_DIR}"
```

Comment:
Maybe we can make the `check_spelling.sh` script executable, and add a hashbang at the top of it, then we don't need to invoke "/bin/bash" manually here.

Reply:
Sure, there's already a hashbang, will update the cmake file.

Reply:
Also not all systems have bash installed in `/bin/bash` and I'd advise to use `/usr/bin/env bash` instead, especially in the shebang line.